.travis/build_package.sh

```diff
@@ -19,7 +19,7 @@ function build_demo {
   cd build-travis
   if [ $NEED_3D = 1 ]; then
     #install libqglviewer
-    git clone --depth=1 https://github.com/GillesDebunne/libQGLViewer.git ./qglviewer
+    git clone --depth=4 -b v2.6.3 --single-branch https://github.com/GillesDebunne/libQGLViewer.git ./qglviewer
```

Comment:
Yes it was intended, because of the problems we had with libQGLViewer 2.7.0. Maxime did the patch in #2186 , and I cherry-picked it in all pull-requests, so that the Travis status of the pull-requests could be green again.